src/main/database/entities/DriveFile.ts

```diff
@@ -0,0 +1,99 @@
+import { DatabaseCollectionAdapter } from '../adapters/base';
+
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+import { AppDataSource } from '../data-source';
+
+@Entity()
+export class DriveFile {
+  @PrimaryColumn({
+    nullable: false,
+  })
+  fileId!: string;
+
+  @Column({ nullable: false })
+  id!: number;
+
+  @Column({ nullable: false })
+  uuid!: string;
+
+  @Column({ nullable: false })
+  type!: string;
+
+  @Column({ nullable: false })
+  size!: number;
+
+  @Column({ nullable: false })
+  bucket!: string;
+
+  @Column({ nullable: false })
+  folderId!: number;
+
+  @Column({ nullable: true })
+  folderUuid?: string;
+
+  @Column({ nullable: false })
+  userId!: number;
+
+  @Column({ nullable: true })
+  modificationTime?: string;
+
+  @Column({ nullable: false })
+  createdAt!: string;
+
+  @Column({ nullable: false })
+  updatedAt!: string;
+
+  @Column({ nullable: true })
+  plainName?: string;
+}
+
+export class DriveFilesCollection
+  implements DatabaseCollectionAdapter<DriveFile>
+{
+  private repository = AppDataSource.getRepository(DriveFile);
+
+  async connect(): Promise<{ success: boolean }> {
+    return {
+      success: true,
+    };
+  }
+
+  async get(fileId: DriveFile['fileId']) {
+    const match = await this.repository.findOneBy({ fileId });
+    return {
+      success: true,
```

Comment:
Why are we returning success: true if we are not catching to return succes: false? Also, we should add the return type here, as the get will or will not found the desired `file`. I suggest using `uuid` as the key to identify files as the autoincremental id will disappear in the future and the `fileId` is a field from the network which is not directly related to Drive and therefore, could change in the future.

Reply:
types are inferred from the base abstract class, and are modified using a TS generic, see here: https://github.com/internxt/drive-desktop/blob/4edc8ce9ad4c31f299a5e7bdf11cd11372eedaad/src/main/database/adapters/base.ts

I moved the primary key to uuid.

About the success true, this operation cannot fail (there's a method called findOneByOrFail that does throw if no entry is found, is not being used here), so this basically means that the operation succeded, but maybe no entry was found in the db, which would result in `result:null` 

I added the success field to unify those adapters that maybe under some circumstances could lead to failed operations, but is not related to the result of the operation itself